pkg/PORTING.md

```diff
@@ -0,0 +1,11 @@
+This directory provides some porting information for libraries and programs to
+use with RIOT (to build an external module). If you'd like to add a package to
+RIOT you need to add a directory with the name of your package to this directory.
+Your directory should contain at least two files:  
+  
+ * patch.txt - Your patch of the upstream application of the package to make
```

Comment:
Can you do it like this?

```md
 * **patch.txt**: Your patch [...]
```

I think the readability would greatly benefit from it.